{{cookiecutter.app_name}}/lib/api/v1/controllers/health_test.go

```diff
@@ -0,0 +1,28 @@
+// SAFETODELETE: This file is an example API testing and can be deleted.
```

Comment:
Actually, we should remove this comment as it's a good practice to keep a health endpoint (especially for Kubernetes deployments). One should not have to delete any file from the template 🧐

Reply:
Removed in db285a00386dd96f47bdc8df8d2540778e8c45bb